cmd/server-manager/plugins/events.lua

```diff
@@ -92,7 +92,8 @@ function weatherAPI(raceConfig, serverOpts, apiKey)
             weather["BaseTemperatureRoad"] = 4
         else
             -- sun is down, base road temp is lower than ambient (large assumption, definitely wrong, please improve!)
-            weather["BaseTemperatureRoad"] = -4
+            -- Mikado : Road can't be colder than the air :)
```

Comment:
Is this an AC limitation? In the real world it definitely can :P

Reply:
Hum... How? Heat is an accumulation. Ground can be 0° if the air is at 0, and remains 0 a moment even if the air is hotter. But it can't become colder just because of the lack of sunshine. Maybe I must find some expert

Reply:
I guess it's very situation dependant, but my understanding is that the air will heat up faster than the ground (specific heat density(?)), so as the air warms up the ground will follow it, but slower. (without any other factors - rain, snow, cars, direct vs indirect sunlight etc.). That also means it cools down slower too though, there are a lot of factors in the real world. Thinking about it I guess that means it'll only ever really be colder at sunrise, so -4 is a bad assumption (unless I'm missing something from when I wrote this).

Having said that for this tiny change I don't see any point in doing actual research to find out - it's probably more sensible to leave it at 0 so they are the same. It doesn't really make a difference anyway, as soon as the cars start driving it'll warm up fast.